Lib/asyncio/queues.py

```diff
@@ -196,7 +253,12 @@ def task_done(self):
 
         Raises ValueError if called more times than there were items placed in
         the queue.
+
+        Raise a QueueShutDown if _shutdown_state is set to
+        `SHUTDOWN_IMMEDIATE`.
```

Comment:
```suggestion
        Raises QueueShutDown if the queue has been shut down immediately.
```

Doc-strings are for humans